extension/manifest.json

```diff
@@ -15,6 +15,7 @@
     "experienceSamplingPrivate",
     "notifications",
     "storage",
+    "tabs",
```

Comment:
Note that this disable existing installations and  cause a "permission increase" prompt to appear. Not that there is anything else you can do though :/